test/Test/Config.hs

```diff
@@ -0,0 +1,26 @@
+module Test.Config (spec) where
+
+import System.IO.Silently (capture)
+import System.IO.Temp (writeSystemTempFile)
+import Test.Hspec
+
+import Hintman.Config
```

Comment:
All imports should be either qualified or have explicit import lists according to our style guide